vm/devices/storage/storvsp/src/lib.rs

```diff
@@ -7,12 +7,9 @@
 #[cfg(feature = "ioperf")]
 pub mod ioperf;
 
-#[cfg(feature = "fuzz_helpers")]
+// Needs to be pub for use with storage tests in a separate crate.
 pub mod test_helpers;
```

Comment:
Please define a cfg feature for test and do that instead. I don't remember, but if this is the only `fuzz_helpers` usage then you can just change the name. I'd rather not make the `test_helpers` public by default.